uwblueprint/planet-read
# Frontend/preview import

## Notion ticket link



[Implement Preview Story Flow on ImportStory page](https://www.notion.so/uwblueprintexecs/Implement-Preview-Story-Flow-on-ImportStory-page-e6c1d92bc65149d5b66ff58db488190f)



## Implementation description

- Added preview modal for import story page



## Steps to test

1. Login as Angela
2. Click on Import Story
3. Verify that you cannot click on Preview Story if no file has been added
4. Add a file, verify that you can only add a docx file
5. Click on Preview Story and verify that the file can be previewed



## What should reviewers focus on?

- Does it work?
- Does the UI look ok?
- Does the code look ok? Also I did some weird git shenanigans so please let me know if anything is broken or wrong!

## Checklist
